source/Tests/Parsing/ParserFixture.cs

```diff
@@ -29,7 +29,7 @@ public void InvalidDocumentReportsCorrectLineNumber()
                     Foo = ""Bar
             ");
 
-            error.Should().Be("Parsing failure: unexpected 'F'; expected end of input (Line 4, Column 21); recently consumed:           ");
+            error.Should().MatchRegex("Parsing failure: unexpected '\r|\n'; expected \" \\(Line 4, Column 31\\); recently consumed: Foo = \"Bar");
```

Comment:
Regarding the `\r|\n`, had to do this to get the test to pass on both my local machine and CI. Initially thought it was Windows/Linux, but could be git. Not sure, either way, this works.

Reply:
This regex does an `or` since the `|` character isn't in brackets, so it would match `Parsing failure: unexpected '\r`.

Reply:
I just don't get why the line endings in a verbatim string literal would change based on the OS. I thought that .net used the line endings encoded into the file itself. Maybe it's teamcity normalising the line endings? Either way, what you've done is much tidier. 👍

Reply:
I think it's because on linux git checks out the code with `\n` and on windows with `\r\n` (i.e the `autoclrf` setting)